optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc

```diff
@@ -69,91 +69,50 @@ To run the back end in development mode, enter the following command:
 
 [source,shell]
 ----
-mvn spring-boot:run
+mvn compile quarkus:dev
 ----
 
-=== Automatic restart
+=== Quarkus development mode
 
-https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-devtools-restart[Automatic restart]
-is provided by Spring Boot DevTools and only works when the back end is running using <<run-spring-boot-maven-plugin, Spring Boot Maven Plugin>>.
-It scans files on the classpath, so you only need to recompile your changes to trigger application restart.
-No IDE configuration is needed.
+In development mode, the backend automatically restarts whenever you refresh the browser tab where the front end runs if there are any changes in the back-end source code or configuration.
 
-If your IDE has a compile-on-save feature (for example Eclipse or NetBeans), you just need to save the files that have changed since the last compilation.
+Learn more about https://quarkus.io/guides/maven-tooling#development-mode[Quarkus development mode].
 
-IntelliJ IDEA saves changes automatically and you need to select either menu:Build[Recompile], which recompiles the file in the active tab, or menu:Build[Build Project] which recompiles all changes.
-See https://www.jetbrains.com/help/idea/compiling-applications.html[Compile and build applications with IntelliJ IDEA].
+=== Running the back end from IntelliJ IDEA Ultimate
 
-=== Running the back end from IntelliJ IDEA
+IntelliJ IDEA Ultimate has a bundled Quarkus plugin that automatically creates run configurations for modules using the Quarkus framework.
```

Comment:
plug-in, not plugin, in all cases. Maybe search for you in your files? (IBM Style Guide)